client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoMemoryResultSet.java

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
why is this class not in core?

Reply:
I needed to add at least one class so that non-empty jar is produced. Adding it to JDBC doesn't make sense as it will be shaded along with trino-client classes so you won't be able to use it.